src/huggingface_hub/hf_api.py

```diff
@@ -1197,12 +1197,58 @@ class User:
             Name of the user on the Hub (unique).
         fullname (`str`):
             User's full name.
+        num_models (`int`, *optional*):
+            Number of models created by the user.
+        num_datasets (`int`, *optional*):
+            Number of datasets created by the user.
+        num_spaces (`int`, *optional*):
+            Number of spaces created by the user.
+        num_discussions (`int`, *optional*):
+            Number of discussions initiated by the user.
+        num_papers (`int`, *optional*):
+            Number of papers authored by the user.
+        num_upvotes (`int`, *optional*):
+            Number of upvotes received by the user.
+        num_likes (`int`, *optional*):
+            Number of likes given by the user.
+        is_following (`bool`, *optional*):
+            Whether the authenticated user is following this user.
+        details (`str`, *optional*):
+            User's details.
     """
 
     # Metadata
     avatar_url: str
     username: str
     fullname: str
+    num_models: Optional[int] = None
```

Comment:
I'm sorry I made a mistake in https://github.com/huggingface/huggingface_hub/pull/2147#discussion_r1541223989. Would it be possible to add back `is_pro: Optional[bool] = None` to the list. I only wanted to select `user_type` as the field to remove. Thanks in advance!

(also update `__init__.py` and docstring accordingly)

Reply:
@Wauplin Fixed it!